src/components/users/TickForm.tsx

```diff
@@ -24,7 +24,8 @@ const TickSchema = Yup.object().shape({
   attemptType: Yup.string()
     .required('Please choose an ascent type'),
   dateClimbed: Yup.date()
-    .required('Please include a date'),
+    .required('Please include a date')
+    .max(new Date(), 'Please include a date in the past'),
```

Comment:
Is the date check done in the user's local timezone?  Looks like it from the comment on L#95?